<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvnCommandDialog</class>
 <widget class="QDialog" name="SvnCommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subversion Command</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Subversion Command:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="commandCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter the Subversion command to be executed with all necessary parameters</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Subversion Command&lt;/b&gt;
&lt;p&gt;Enter the Subversion command to be executed including all necessary 
parameters. If a parameter of the commandline includes a space you have to 
surround this parameter by single or double quotes. Do not include the name 
of the subversion client executable (i.e. svn).&lt;/p&gt;</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="workdirCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter the working directory for the Subversion command</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Working directory&lt;/b&gt;
&lt;p&gt;Enter the working directory for the Subversion command.
This is an optional entry. The button to the right will open a 
directory selection dialog.&lt;/p&gt;</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel2">
       <property name="text">
        <string>Working Directory:&lt;br&gt;(optional)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Project Directory:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="projectDirLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>This shows the root directory of the current project.</string>
       </property>
       <property name="text">
        <string>project directory</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="dirButton">
       <property name="toolTip">
        <string>Select the working directory via a directory selection dialog</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Working directory&lt;/b&gt;
&lt;p&gt;Select the working directory for the Subversion command via a directory selection dialog.&lt;/p&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>commandCombo</tabstop>
  <tabstop>workdirCombo</tabstop>
  <tabstop>dirButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SvnCommandDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SvnCommandDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
